README.md

```diff
@@ -430,6 +376,19 @@ extensions are added automatically.
 + `max-file-size`: optional parameter to set the appromximate maximum file size in MB of the resulting files (default: 25)
 + `delete-originals`: optional, default False.
 
+#### Reading Parquet files
+If you are using Python, you can read the Parquet files into a Pandas or Polars DataFrame, 
+or use the [pyarrow](https://pypi.org/project/pyarrow/) package.
+For R you can use the [arrow](https://arrow.apache.org/docs/r/reference/read_parquet.html) package.
+
+Each Parquet-file contains a number of rows, each one corresponding with one scraped URL, with the following columns:
++ job_tag: id.
++ domain: domain for which the CDX-function retrieved the URL.
++ url: full URL that was scraped.
++ page_text: full page text
++ page_links: list of page links
++ timestamp: timestamp of creation of the record
+
 
 ## Cleaning up
 ### Destroying the infrastructure
```

Comment:
Maybe 'Destroying' is a little strong; how about 'Deleting'?

Reply:
You are right :-) Replaced destroying with deleting